app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -175,6 +176,15 @@ class SupplementalApplicationPage extends React.Component {
     }
   }
 
+  handleClickCreateLease = () => {
+    this.setState({ showLeaseSection: true })
+  }
+
+  handleClickCancelLease = () => {
```

Comment:
nit: Remember when I said the new version of Standard has weird rules around `handle*` and `on*`naming? This reminded me I didn't really understand it so I googled it and found this: https://jaketrent.com/post/naming-event-handlers-react/

Basically the function should be named `handle*` and the props should be `on*`. I think that kind of makes sense and reads nicely in the component:
```
<button onClick={handleButtonClick} />
```

So in this case we'd keep the function name and the context variable `handleClickCancelLease`/`handleClickCreateLease`, but the prop names would change.

It looks like we're already using `onCancel` for the cancel case so that's good, but maybe we can change the `LeaseSection` prop to `onClickCreate` or something?

Reply:
I just went to update then got confused. What happens when it's a function that you're passing as a prop down many levels? Am I just overthinking it?

Reply:
@akegan yeah, I thought the same thing. I think only the top level would be handle-, all of the props are on-

Basically, if it's a function, it should be called handle-, if it's a prop it should be on-

(at least according to what that one blog post said that we can totally ignore if you want to)

Reply:
OK just pushed changes based on the article. Also renamed to put click at the end because i liked that suggestion too. Let me know if that's what you were thinking! Still feels kinda weird and inconsistent.